### ISSUE TITLE
Players get kicked when fighting Rubro

### ISSUE THREAD

**Issue by xErrorAmelie** [Klyser8/Earthbounds#20]
https://paste.corebot.dev/enaxotasic.php
This happend several times now. It happens not everytime but many times.
In the last two cases Rubro got shot by a whiter skull, in the second case by a arrow.

The player who shot him gets kicked with the provided error and can rejoin after that.

**Comment by Klyser8** [comment]
Sorry for the late reply, I was on holiday!

What's confusing about this error is that I have encountered it before very rarely, and I always struggle to replicate it as there's no code pointing to my mod. Has this error happened again since you've written this?

**Comment by xErrorAmelie** [comment]
Yep, still happens every time I shoot him. It would wonder me if you can't reproduce it. Maybe I have to say that the modpack where I play is Another Quality Modpack 2.

**Comment by Klyser8** [comment]
Is this resolved in the latest Earthbounds version?

**Comment by xErrorAmelie** [comment]
It's still there. Here are full server logs and a video x)
https://mclo.gs/DyGgUUX
https://www.youtube.com/watch?v=ZkW8axnrUhw

**Comment by Klyser8** [comment]
Alright, I'll look into it! P.S. Do you have any shaders enabled? They seem to be messing with the Rubro's model and lighting.

**Comment by xErrorAmelie** [comment]
No, didn't use shaders.

**Comment by MScott7** [comment]
I think I've been having this issue too. It's hard to tell though since the error messages don't point to a specific mod but my logs look extremely similar to yours. 

Logs: https://pastebin.com/BSzqYrVq
If it helps, I'm on Minecraft 1.18.2, Fabric 0.14.8, Fabric API version 0.58.0, Earthbounds version 1.1.4, Geckolib version 3.0.52.
